src/components/spatial/Spatial.js

```diff
@@ -71,6 +78,30 @@ export default class Spatial extends AbstractSelectableComponent {
     return this.props.view;
   }
 
+  componentWillMount() {
+    this.slidersChangeToken = PubSub.subscribe(SLIDERS_CHANGE, this.onSlidersChange.bind(this));
+    this.colorsChangeToken = PubSub.subscribe(COLORS_CHANGE, this.onColorsChange.bind(this));
+    this.channelsToggleToken = PubSub.subscribe(CHANNEL_TOGGLE, this.onChannelsToggle.bind(this));
```

Comment:
CHANNEL_TOGGLE vs this.onChannel<b>s</b>Toggle

I don't care which, but we should be consistent.